src/python/benchUtil.py

```diff
@@ -140,7 +140,7 @@ def verifySame(self, other, verifyScores, verifyCounts):
           print 'WARNING: expandedTermCounts differ for %s: %s vs %s' % (self, self.expandedTermCount, other.expandedTermCount)
           # self.fail('wrong expandedTermCount: %s vs %s' % (self.expandedTermCount, other.expandedTermCount))
 
-      if verifyCounts and self.hitCount != other.hitCount:
+      if False and self.hitCount != other.hitCount:
```

Comment:
Woops, was this accidental?  Maybe add `# nocommit` so we don't accidentally break this?

Reply:
Removed in f87f287ddc50aeec97fdf0379d5a2ba1a47b75dc

Thank you for the review!